### PR TITLE
UGENE-7701 [Wd, crash] Set breakpoint to last element, run workflow, close window

### PR DESCRIPTION
**Pull request by rasputinkirill**
В слотах WorkflowIterationRunTask нет нужды проверять debugInfo на nullptr так как при его деструкции они будут уже отключены.

### CODE REVIEW

File: src/corelibs/U2Lang/src/support/WorkflowRunTask.cpp [hunk 1]
```diff
@@ -339,7 +342,7 @@ Task::ReportResult WorkflowIterationRunTask::report() {
 }
 
 WorkerState WorkflowIterationRunTask::getState(const ActorId& id) {
-    if (scheduler) {
+    if (scheduler && debugInfo) {
```
Comment (yalgaer) [inline src/corelibs/U2Lang/src/support/WorkflowRunTask.cpp]: Don't you need to move debug info into the inner check?
```
debugInfo && debugInfo->isPaused()
```

Comment (rasputinkirill) [reply src/corelibs/U2Lang/src/support/WorkflowRunTask.cpp]: 👍

**Comment by yalgaer** [comment]
Looks good. Will approve in the current state after the tests are green

**Comment by yalgaer** [comment]
Closing as not active. Please re-open when tests are fixed

**Pull request closed without merging**